src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/BasePlatformBehavior.shared.cs

```diff
@@ -0,0 +1,75 @@
+#if IOS || MACOS || MACCATALYST
+using PlatformView = UIKit.UIView;
+#elif ANDROID
+using PlatformView = Android.Views.View;
+#elif WINDOWS
+using PlatformView = Microsoft.UI.Xaml.FrameworkElement;
+#elif TIZEN
+using PlatformView = Tizen.NUI.BaseComponents.View;
+#elif NET6_0_OR_GREATER || (NETSTANDARD || !PLATFORM)
+using PlatformView = System.Object;
+#endif
+
+using System.ComponentModel;
+
+namespace CommunityToolkit.Maui.Behaviors;
+
+
+/// <summary>
+/// Abstract class for our behaviors to inherit.
+/// </summary>
+/// <typeparam name="TView">The <see cref="VisualElement"/> that the behavior can be applied to</typeparam>
+public abstract partial class BasePlatformBehavior<TView> : BasePlatformBehavior<TView, PlatformView>
```

Comment:
We need this member to be `public` in order to use on our `public` Behaviors, but do we want external assemblies to create it? As you mentioned on discussion we would like to break the .NET MAUI default behavior outside our lib, and I agree with that. We're setting the `BindingContext` here, but we _know_ what we're doing, could be dangerous to expose it to the world, so my suggestion is to add an `internal ctor` for those `abstract` classes.

Reply:
Done ✅